quantum/rgb_matrix/rgb_matrix_types.h

```diff
@@ -83,7 +83,7 @@ typedef struct PACKED {
 } led_config_t;
 
 typedef union {
-    uint32_t raw;
+    uint64_t raw;
     struct PACKED {
         uint8_t     enable : 2;
         uint8_t     mode : 6;
```

Comment:
Cant mark as suggestion but line 91 `// EECONFIG needs to be increased to support this` should be removed